utilities/build_julia.sh

```diff
@@ -10,6 +10,10 @@ set -euo pipefail
 source .buildkite/utilities/build_envs.sh
 source .buildkite/utilities/word.sh
 
+echo "--- Get kernel/compiler versions"
+uname -a
+cc -v
+ld -v
```

Comment:
Would be nice to have a blank line after this line and before the next section starts.

Reply:
You mean like print a new line? Because in the log they are in a different section already

Reply:
No, just a blank line in the shell script, just for readability.

Reply:
Oh :), I was looking at the wrong script and was confused because it already had a newline.